app/models/miq_server/log_management.rb

```diff
@@ -132,6 +134,26 @@ def post_logs(options)
     task.update_status("Finished", "Ok", "Log files were successfully collected")
   end
 
+  def current_log_pattern_configuration
+    get_config("vmdb").config.fetch_path(:log, :collection, :current, :pattern) || []
+  end
+
+  def current_log_patterns
+    current_log_pattern_configuration | pg_log_patterns
```

Comment:
Pull in the current pattern from the configuration and add the pg_log_patterns if the configuration didn't already have them.